src/state/slices/txHistorySlice/txHistorySlice.test.ts

```diff
@@ -197,40 +195,4 @@ describe('txHistorySlice', () => {
       ).toStrictEqual([serializeTxIndex(segwitAccountId, BtcSendSegwit.txid, BtcSendSegwit.pubkey)])
     })
   })
-
-  describe('selectLastNTxIds', () => {
-    it('should memoize', () => {
-      const txs: TxsState = {
-        byId: {},
-        byAccountIdAssetId: {},
-        ids: ['a', 'b'],
-      }
-
-      const txHistory: TxHistory = { txs }
-
-      const state = {
-        ...mockStore,
-        txHistory,
-      }
-      const first = selectLastNTxIds(state, 1)
-
-      const newTxHistory: TxHistory = {
-        txs: {
-          ...txs,
-          // this array will always change on every new tx
-          ids: ['a', 'b', 'c'],
-        },
-      }
-
-      // redux will replace the array on update
-      const newState = {
-        ...mockStore,
-        txHistory: newTxHistory,
-      }
-      const second = selectLastNTxIds(newState, 1)
-
-      // toBe uses reference equality, not like isEqual deep equal check
-      expect(first).toBe(second)
-    })
-  })
```

Comment:
This test is no longer relevant, as we're using cached selectors which fire based on changes to input values and filter parameters, rather than the transaction history being cached based on serialized output value equality